stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageContainer.kt

```diff
@@ -58,6 +60,8 @@ import io.getstream.chat.android.ui.common.state.messages.list.TypingItemState
  * @param systemMessageContent Composable that represents system messages.
  * @param messageItemContent Composable that represents regular messages.
  * @param typingIndicatorContent Composable that represents a typing indicator.
+ * @param emptyThreadPlaceholderItemContent Composable that represents placeholders inside of an empty thread.
+ * This content is disabled by default and can be enabled via [MessageListController.showDateSeparatorInEmptyThread].
```

Comment:
Maybe add `MessageListViewModel` or `MessageListViewModelFactory`. Users using our default components will probably create the VM through them. Users using `MessageListControler` will probably get that from it's docs.

Reply:
Updated